integration/jena.md

```
@@ -0,0 +1,84 @@
+If the adapter is published to a non-standard Maven repository, add the repository block shown in the adapter's README. Otherwise the dependency should resolve from Maven Central (if published there).
+
+### Quick Getting Started (Java)
+
+The snippet below shows a minimal Jena program that registers the FalkorDB adapter and runs a simple SPARQL query. This is intentionally small — see the adapter's `GETTING_STARTED.md` and `Main.java` for a complete example with configuration and connection options.
+
+```java
+import org.apache.jena.query.*;
+import org.apache.jena.rdf.model.Model;
+
+public class JenaFalkorExample {
+	public static void main(String[] args) {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)